Add fastify-cockroachdb to Community plugins

alex-ppg opened the pull request:
Greetings everyone, 

I created this plugin that injects a Sequelize instance connected to the PostgreSQL database of CockroachDB along with optional pre-defined models within `fastify`. Defining models is optional and can be done post-injection by accessing `fastify.cockroachdb.instance`.

I recently required it for a project and thought that others might find it useful.

Thanks.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

jsumners commented:
I don't think we should attempt to restrict the licensing of community modules. Officially maintained ones, yes, but not community.

alex-ppg commented:
@allevo Added `'use strict'` which was missing in the `test.js` file. Adding a CI tool would require hefty customization as running a CockroachDB cluster does not exist by default on Travis or the common CI tools. If it is **required**, I can get that done and re-open (?) this pull request.

alex-ppg commented:
@mcollina As requested, I have added a CircleCI configuration .yml file to the original repository and also added the CircleCI badge to the Github page.
